docs/gff3_sort.md

```diff
@@ -32,8 +32,6 @@ Sort features in a gff3 file by according to their order on a scaffold, their co
     - Sort multi-isoform gene models by feature type (default: False)
 6. -v, --version
     - show program's version number and exit
-7. -r, --reference
```

Comment:
Can you remind me where is this removal comes from ?

Reply:
@ShangYuChiang I wonder if you were reviewing an older version of gff3_sort? Tony recently pulled an update into master that added the --reference argument. That code is also missing in your branch of gff3_sort.py...